### PR TITLE
Add sponsor

### PR DESCRIPTION
**Pull request by nlin21**

### CODE REVIEW

File: _data/sponsors/sponsors.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@
   site: http://www.bloomberg.com/
 - name: Solidworks
   logo: https://stuypulse.nyc3.cdn.digitaloceanspaces.com/site/sponsorships/img/solidworks.png
-  site: http://www.solidworks.com/
```
Comment (yuna9) [inline _data/sponsors/sponsors.yml:28]: Remove this change for now, we'll solve it in a differnt changeset

File: kickoff/index.md [hunk 2]
```diff
@@ -7,7 +7,6 @@ title: 2020 Kickoff Information
 <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js" integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1" crossorigin="anonymous"></script>
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-<link rel="stylesheet" href="css/styles.css">
```
Comment (yuna9) [inline kickoff/index.md:10]: ?

File: resources/stuysplash2015/index.md [hunk 3]
```diff
@@ -34,4 +34,4 @@ In 2015, we held Stuy Splash at Stuyvesant High School on December 12. Below are
...
+> Over the course of this lecture I will go over the basics of computer assisted design in Solidworks. Solidworks is a program that allows you to design and build 3D models of parts, mechanisms, and assemblies. Over the course of the lecture I will go over the basics of designing parts, putting them together, and running simulated stress tests on them. Solidworks is willing to sponsor any FIRST team that asks for a FIRST sponsorship and will provide Solidworks licenses as necessary depending on the size of your team. If you would like to follow along with the lecture, feel free to bring a laptop that has Solidworks installed.
```
Comment (yuna9) [inline resources/stuysplash2015/index.md:37]: ?

File: kickoff/index.md (hunk 2 above)
Comment (nlin21) [reply kickoff/index.md:10]: The build log returned that the "styles.css" file in the css directory doesn't exist (which it doesn't)
I figured if the website works without the css file I could just delete it

File: resources/stuysplash2015/index.md (hunk 3 above)
Comment (nlin21) [reply resources/stuysplash2015/index.md:37]: The FIRST Solidworks Sponsorship link doesn't exist anymore and I couldn't find an appropriate replacement for it

**Pull request merged**